### DISCUSSION TITLE
Can you use a third party library for auth and use pocketbase at the same time

### DISCUSSION THREAD

**Discussion by Ebibeck** [pocketbase/pocketbase#3340] [Q&A]
Can you use a third party library for auth and use pocketbase at the same time in next js specifically with next auth

**Comment by ganigeorgiev** [comment]
_I assume you mean [NextAuth.js](https://next-auth.js.org/)._
I'm not familiar with NextAuth but the integration would certainly not work out of the box.

But I'm failing also to understand what are your expectations and the need of the integration.
If you just want to use SQLite you don't really need PocketBase in this case and can interact directly with your database via node.

Also note that SSR with Next.js at the moment is kindof difficult to integrate with PocketBase in general due to the lack of easy way to pass server-side data between components and middlewares and I'd advice to reevaluate whether you need the complexity of maintaining 2 server-side backends in the first place.

**Reply by Ebibeck** [reply]
yikes
